sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/LiveMetrics/DataCollection/DocumentHelper.cs

```diff
@@ -191,7 +191,15 @@ internal static Request ConvertToRequestDocument(Activity activity)
                 }
                 else if (tag.Key == SemanticConventions.AttributeServerPort)
                 {
-                    serverPort = tag.Value.ToString()!;
+                    // The Instrumentation.AspNetCore library sets "server.port" as an object{int}. Doing the same here for comparison to avoid unboxing.
+                    // https://github.com/open-telemetry/opentelemetry-dotnet/blob/Instrumentation.AspNetCore-1.7.0/src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs#L187-L190
+                    object port80 = 80;
+                    object port443 = 443;
+
+                    if (!tag.Value.Equals(port80) && !tag.Value.Equals(port443))
```

Comment:
It's not common to check object equals? Was it recommended anywhere?

Reply:
No need to use `ToString()` with `{}`

Reply:
i was trying to avoid the unboxing. i'll make the change.